blueflood-http/src/main/java/com/rackspacecloud/blueflood/http/DefaultHandler.java

```diff
@@ -18,23 +18,60 @@
 
 import com.codahale.metrics.Timer;
 import com.rackspacecloud.blueflood.io.Constants;
+import com.rackspacecloud.blueflood.outputs.formats.ErrorResponse;
 import com.rackspacecloud.blueflood.tracker.Tracker;
 import com.rackspacecloud.blueflood.utils.Metrics;
 import io.netty.buffer.Unpooled;
 import io.netty.channel.ChannelHandlerContext;
 import io.netty.handler.codec.http.*;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import java.io.IOException;
 import java.util.Iterator;
+import java.util.List;
 import java.util.Map;
 
 public class DefaultHandler implements HttpRequestHandler {
     private static final Timer sendResponseTimer = Metrics.timer(DefaultHandler.class, "HTTP response sending timer");
 
+    private static final Logger log = LoggerFactory.getLogger(DefaultHandler.class);
+
     @Override
     public void handle(ChannelHandlerContext ctx, FullHttpRequest request) {
         HttpResponder.respond(ctx, request, HttpResponseStatus.OK);
     }
 
+
+    private static String prepareErrorResponse(String tenantId, String message) {
+        return "{\"errors\": [{\"tenantId\": \"" + tenantId + "\", \"message\": \"" + message + "\"}]}";
```

Comment:
Do we need to worry about escaping the message? What if it contains quote or single quote or curlies?